### ISSUE TITLE
NVDA steals keybindings for moving cursors

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52679]
**Environment Details:**
VSCode Version : 1.24.0
OS Version : Win10

**Additional Details:**
MAS Violated: MAS 2.1.1

**Repro Steps:**
1. Launch VS Code.
2. Open VS Code Editor window.
3. Type some code within the editor and try commands for "Add Cursor Above/Below" and "Add Cursor To Line Ends".

**Actual:**
Keyboard shortcuts for : 
"Add Cursor Above" - Ctrl + Alt + UpArrow 
"Add Cursor Below" - Ctrl + Alt + DownArrow 
"Add Cursor To Line Ends" - Shift + Alt + I
are not working in editor.

**Expected:**
Keyboard shortcuts should work for the above mentioned commands.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
User will not be able to use possible shortcuts listed in the application that might disrupt his working.

**MAS Reference:**
MAS 2.1.1 - https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

**Attachment For Reference:**
[A11y_VSCode_Editor_Keyboard_ShortcutsNotWorking.pptx](https://github.com/Microsoft/vscode/files/2127590/A11y_VSCode_Editor_Keyboard_ShortcutsNotWorking.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
These keybindings are stolen by NVDA. A user can configure different keybindings for those actions.
There is nothing we can do here imho, but leaving to @alexandrudima to decide

**Comment by alexandrudima** [comment]
No action for us here.
